packages/tablev2/Table.tsx

```diff
@@ -0,0 +1,340 @@
+import * as React from "react";
+import { cx } from "emotion";
+import * as style from "./style";
+import { typedMemo } from "../../typings/typedMemo";
+import { CheckboxCell, Draggable, HeaderCellContent } from "./Util";
+import { ColumnProps } from "./Column";
+import { ResetButton } from "../button";
+
+type ColumnChild<A, B> = React.ReactElement<ColumnProps<A, B>>;
+
+// TODO: make it possible to disable rows from being selected with a checkbox
```

Comment:
I'm not sure if we need this for the MVP of Table V2. Reviewer/s: would you be comfortable with creating a follow-up Jira ticket for this? Or do you think it should be done now?

Reply:
totally not a priority for dcos-ui.